AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -189,7 +189,9 @@ - (void)_updateVisibleNodeIndexPaths
   
   ASInterfaceState selfInterfaceState = [self interfaceState];
   ASLayoutRangeMode rangeMode = _currentRangeMode;
-  if (!_didUpdateCurrentRange) {
+  // If the range mode is explicitly set via updateCurrentRangeWithMode: it will last in that mode until the
+  // range controller becomes visible again or explicitly changes the range mode again
+  if (!_didUpdateCurrentRange || ASInterfaceStateIncludesVisible(selfInterfaceState)) {
```

Comment:
I think this may need to be &&?  What if we change the range mode but are still visible?  I think we want that to stick for at least one pass.